app/models/user-model.coffee

```diff
@@ -36,12 +36,12 @@ global.cobudgetApp.factory 'UserModel', (BaseModel) ->
       @groups()[0]
 
     isMemberOf: (group) ->
-      !!_.find @memberships(), (membership) ->
-        membership.groupId == group.id
+      @isSuperAdmin || (!!_.find @memberships(), (membership) ->
+        membership.groupId == group.id)
 
     isAdminOf: (group) ->
-      !!_.find @memberships(), (membership) ->
-        membership.groupId == group.id && membership.isAdmin
+      @isSuperAdmin || (!!_.find @memberships(), (membership) ->
+        membership.groupId == group.id && membership.isAdmin)
 
```

Comment:
TBH I am not sure 100% sure if these are needed. I added them before realizing that we needed to actually fake the membership to get the group data from the backend. so perhaps we can get rid of this